go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/lazy-stripes/goholint
+
+go 1.16
```

Comment:
Can we just lower this to 1.13? This is the version I'm using on Ubuntu, Goholint should work with it.

Reply:
Done.